### PR TITLE
Avoid the term whitelist

### PR DESCRIPTION
**Pull request by nkiesel**
This term was replaced with allowedList. See e.g. https://tools.ietf.org/id/draft-knodel-terminology-00.html
for a rationale.

### CODE REVIEW

**Comment by drei01** [comment]
This looks good to me, thanks for picking up on my original request.

**Comment by drei01** [comment]
Any updates on this PR?

**Comment by jhy** [comment]
Thanks, will plan to get this in for 1.14.1.

File: src/test/java/org/jsoup/safety/CleanerDeprecatedTest.java [hunk 1]
```diff
@@ -0,0 +1,331 @@
+package org.jsoup.safety;
```
Comment (jhy) [inline src/test/java/org/jsoup/safety/CleanerDeprecatedTest.java:1]: Per other test comment, this should not be an entire redundant copy. Just the deprecated names.

**Review by jhy — CHANGES_REQUESTED** [review]
Thanks for this - general approach is good but please see notes around preserving the file change history, and not duplicating the entire test files redundantly.

**Comment by jhy** [comment]
I am happy to make the noted changes BTW, LMK.

**Comment by jhy** [comment]
Thanks - rolled this mostly in to #1464 with a few minor changes (called it Safelist, mainly)

**Pull request closed without merging**